### ISSUE TITLE
How can I connect to 'flask'?

### ISSUE THREAD

**Issue by Lee-daeho** [vasturiano/react-force-graph#254]
**Describe the bug**

Hi, I'm trying to connect the html code to flask(python).
When I load html only by IDE(Pycharm), it makes a proper result. 
But, when I load it by flask, the result says **'Uncaught (in promise) SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data'**

Is there any possible way to connect it to flask? I need it because I have to 'execute' the html file with arguments.

Thank you

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/54104389/108682120-4537d180-7533-11eb-8e88-b8f6c87abf63.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 18.04
 - Browser [e.g. chrome, safari] FireFox

**Smartphone (please complete the following information):**

**Additional context**

**Comment by vasturiano** [comment]
@Lee-daeho it's difficult to say without more context, but it appears the URL you're trying to load for the graph may be returning something else other than a JSON response, causing the parser to choke.

I would look in the network tab to see if the data is being fetched correctly.

**Comment by Lee-daeho** [comment]
Thanks for fast reply.

This is my fetch result. Fetch fail to load json.

![image](https://user-images.githubusercontent.com/54104389/108803368-db283680-75dd-11eb-860f-aa5ff70c310f.png)

I think url can't match the storage in local.

Is there any possible solution?

**Comment by vasturiano** [comment]
@Lee-daeho this goes beyond the scope of this module and it's instead related to how your app is setup, but it looks like you don't have any resource behind that URL. See 404 status (not found).

**Comment by Lee-daeho** [comment]
@vasturiano You are right. I think localhost:5000 is not connected to my local storage(somewhere /virtual is defined).

But, when I load the HTML file only, URL is connected well.
